src/ClashOfClans/Models/WarLogEntry.cs

```diff
@@ -10,8 +10,8 @@ public class WarLogEntry
 
         public int? TeamSize { get; set; }
 
-        public WarClan Clan { get; set; }
+        public CurrentWarClan Clan { get; set; }
```

Comment:
The used naming convention follows official Clash of Clans API naming convention of the Model the API provides. In the official API clan and opponent are both WarClan's thus I'm not going to change these to something else.